src/generate/animationdata.cpp

```diff
@@ -1,867 +1,903 @@
+#include "Global.h"
+
 #include "generate/animationdata.h"
 #include "generate/animationdatatracker.h"
 
-#pragma warning(disable:4503)
+#pragma warning(disable : 4503)
 
 using namespace std;
 
 struct InfoDataTracker
 {
-	string data;
-	unsigned int index;
+    string data;
+    uint index;
 
-	InfoDataTracker(string d, unsigned int i) : data(d), index(i) {}
+    InfoDataTracker(string d, uint i)
+        : data(d)
+        , index(i)
+    {}
 };
 
-bool IDExistProcess(string change, vector<InfoDataPack>& storeline, map<string, vector<InfoDataTracker>> original, map<string, bool>& isExist,
-	map<string, string>& exchange, vector<AnimDataPack>& animDataPack, map<string, vector<int>>& codeTracker, map<string, bool>& loopCheck);
-void BehaviorListProcess(AnimDataProject& storeline, int& startline, vecstr& animdatafile, string project, string modcode);
-void AnimDataProcess(vector<AnimDataPack>& storeline, int& startline, vecstr& animdatafile, string project, string modcode, map<string, string>& exchange,
-	map<string, vector<shared_ptr<AnimationDataTracker>>>& animDataTracker, map<string, vector<int>>& codeTracker);
-void InfoDataProcess(vector<InfoDataPack>& storeline, int& startline, vecstr& animdatafile, string project, string modcode, map<string, string>& exchange,
-	vector<AnimDataPack>& animDataPack, map<string, vector<int>>& codeTracker);
-
-bool IDExistProcess(string change, vector<InfoDataPack>& storeline, map<string, vector<InfoDataTracker>> original, map<string, bool>& isExist,
-	map<string, string>& exchange, vector<AnimDataPack>& animDataPack, map<string, vector<int>>& codeTracker, map<string, bool>& loopCheck)
+bool IDExistProcess(string change,
+                    vector<InfoDataPack>& storeline,
+                    map<string, vector<InfoDataTracker>> original,
+                    map<string, bool>& isExist,
+                    map<string, string>& exchange,
+                    vector<AnimDataPack>& animDataPack,
+                    map<string, vector<int>>& codeTracker,
+                    map<string, bool>& loopCheck);
+void BehaviorListProcess(
+    AnimDataProject& storeline, int& startline, VecStr& animdatafile, string project, string modcode);
```

Comment:
shouldn't it be `void BehaviorListProcess(AnimDataProject& storeline`?

Reply:
Yes, it's just misplaced